modules/tabs-v2/module.php

```diff
@@ -9,15 +10,15 @@
 
 class Module extends \Elementor\Core\Base\Module {
 
-	public static function is_active() {
-		return Plugin::$instance->experiments->is_feature_active( 'nested-elements' );
+	public static function is_active(){
```

Comment:
Why delete the space? There should be spaces

Reply:
?